<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactDialog</class>
 <widget class="QDialog" name="ContactDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Contact</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" colspan="3">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="CName">
          <property name="text">
           <string>Name</string>
          </property>
          <property name="buddy">
           <cstring>ContactNAME</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="ContactADDRESS"/>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="ContactNAME"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="CNick">
          <property name="text">
           <string>NickName</string>
          </property>
          <property name="buddy">
           <cstring>ContactNICKNAME</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="ContactNICKNAME"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="CAddress">
          <property name="text">
           <string>Address</string>
          </property>
          <property name="buddy">
           <cstring>ContactADDRESS</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="CPhoneNum">
          <property name="text">
           <string>PhoneNumber</string>
          </property>
          <property name="buddy">
           <cstring>ContactPHONENUM</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="ContactPHONENUM"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>715</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="ContactADD">
        <property name="text">
         <string>ADD</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="ContactEXIT">
        <property name="text">
         <string>EXIT</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ContactNAME</tabstop>
  <tabstop>ContactNICKNAME</tabstop>
  <tabstop>ContactPHONENUM</tabstop>
  <tabstop>ContactADDRESS</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
